deck/views.py

```diff
@@ -47,7 +51,7 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class CreateEvent(BaseEventView, CreateView, FormValidRedirectMixing):
+class CreateEvent(LoginRequiredMixing, BaseEventView, CreateView, FormValidRedirectMixing):
```

Comment:
This line cannot have more than 80 chars, can you break it down?
Something like: 
```py
class CreateEvent(LoginRequiredMixing,
                  BaseEventView,
                  CreateView,
                  FormValidRedirectMixing):
```